### DISCUSSION TITLE
Positioning a copyright statement below all other content in a Quarto manuscript

### DISCUSSION THREAD

**Discussion by tiernanmartin** [quarto-dev/quarto-cli#9115] [Q&A] [answered]
### Description

I want to add a copyright statement to a Quarto manuscript. I do not see a way to include a footer in the YAML metadata, so I have pasted my copyright statement at the end of the document, as the image below illustrates. However, this approach doesn't solve my problem: the Citation section automatically appears at the document's end, pushing the copyright statement up to an awkward position that's not at the very bottom of the page.

How can I make sure my copyright statement remains at the bottom of the document, even with the Citation section included?

![image](https://github.com/quarto-dev/quarto-cli/assets/8197328/bc637af5-dd8d-479c-a4dc-a8096c823638)

<details><summary>Quarto Version & OS</summary>

Quarto version: 1.4.549
OS: Windows 11

</details>

Note: this question was originally posted on Posit Community: 
https://community.rstudio.com/t/positioning-copyright-statement-below-automatic-citation-section-in-quarto-document/181887

**Comment by mcanouil** [comment, marked as answer]
There are different ways depending on the output format as they can have footer (e.g., via CSS for HTML, via special commands for LaTeX/Typst).
Another way, would be to use `include-after-body` to insert content.

See:
- https://github.com/quarto-dev/quarto-cli/discussions/8677#discussioncomment-8473144
- https://github.com/quarto-dev/quarto-cli/discussions/8194

Amongst other threads discussing footer.

**Comment by tiernanmartin** [comment]
Thanks, @mcanouil . These links were helpful.

Here is what I added to my YAML header to create my desired copyright statement:

``` r
---
title: "Manuscript Title"
include-in-header:
  text: |
    <style type="text/css">
    .footer {
      font-size: 0.85em;
      left: 0;
      bottom: 0;
      width: 100%;
      text-align: center;
      padding: 20px;
      background-color: #f2f2f2; /* Light grey background */
      color: #666; /* Dark grey text */
    }
    </style>
include-after-body:
  text: |
    <footer class="footer">Copyright © March 2024. Futurewise. Permission is granted for reproduction of this file, with attribution to Futurewise.
    </footer>
---
# Manuscript

```

Here's a screenshot of how it looks:

<img width="646" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8197328/12b575b1-476e-4c36-8ac3-f7c158356066">
